src/commands/UnbanBanCommand.ts

```diff
@@ -50,7 +50,7 @@ export async function parseArguments(roomId: string, event: any, mjolnir: Mjolni
     while (argumentIndex < 7 && argumentIndex < parts.length) {
         const arg = parts[argumentIndex++];
         if (!arg) break;
-        if (["user", "room", "server"].includes(arg.toLowerCase())) {
+        if (["user", "room", "server", "recentusers"].includes(arg.toLowerCase())) {
```

Comment:
I think it's probably best not to include unrelated changes in the same PR. If problems come up, it's harder to see what caused them, and to back out any problematic changes.

(Also, I don't understand what this is doing)

Reply:
No clue, looks like a merge error.